src/engine/test/source/builder/builders/opBuilderHelperRenameField_test.cpp

```diff
@@ -0,0 +1,103 @@
+#include <any>
+#include <gtest/gtest.h>
+#include <vector>
+
+#include <baseTypes.hpp>
+
+#include "opBuilderHelperMap.hpp"
+
+using namespace base;
+using namespace builder::internals::builders;
+
+using std::string;
+using std::vector;
+
+base::EngineOp op {};
+
+TEST(opBuilderHelperRenameField, build)
```

Comment:
what if the source field is an array, or a nested object, or it has a null or empty value? are we checking those cases?